src/Propel/Generator/Model/ScopedMappingModel.php

```diff
@@ -119,6 +125,22 @@ public function isAbsoluteNamespace($namespace)
         return strpos($namespace, '\\') === 0;
     }
 
+    /**
+     * Prepends a backslash to a namespace if there is none.
+     *
+     * A namespace with a backslash is considered absolute.
+     *
+     * @param string|null $namespace
+     *
+     * @return string|null
+     */
+    public static function makeNamespaceAbsolute(?string $namespace): ?string
+    {
+        $prependBackslash = ($namespace && substr($namespace, 0, 1) !== '\\');
+
+        return ($prependBackslash) ? "\\$namespace" : $namespace;
```

Comment:
Unnecessary parentheses or even variable.

Reply:
The variable is there to keep the return statement readable and to self-comment on what is being checked.
The parentheses are there to highlight that this is an expression, not a simple assignment.

Reply:
I said about `($prependBackslash)`

Reply:
Oh, those parentheses are there to make it easy to see that this is a ternary condition. And, same as above, I think it is helpful to emphasize that this is an expression.
I am well aware that I could cram the whole thing into a single line and omit the parens, but I genuinely consider that bad style. Granted, the kind of bad style that you can live with when working with other programmers.